src/trix/models/block.coffee

```diff
@@ -137,13 +143,13 @@ class Trix.Block extends Trix.Object
     @attributes[depth]
 
   canBeGroupedWith: (otherBlock, depth) ->
-    attributes = @attributes
     otherAttributes = otherBlock.getAttributes()
-    if attributes[depth] is otherAttributes[depth]
-      if attributes[depth] in ["bullet", "number"] and otherAttributes[depth + 1] not in ["bulletList", "numberList"]
-        false
-      else
-        true
+    otherAttribute = otherAttributes[depth]
+    attribute = @attributes[depth]
+
+    attribute is otherAttribute and
+      not (Trix.config.blockAttributes[attribute].group is false and
```

Comment:
@sstephenson - if you're referring to `Trix.config.blockAttributes[attribute].group`, (I think) `getBlockAttributeNames` only returns the keys, so `getBlockAttributeNames()[attribute].group` would be undefined.

I agree though creating and using a helper for text and block attribute names would be more consistent though, so I can use that where possible.

Were you suggesting we make helpers for accessing the keys, as well as the entire attribute object? ie:

```
getBlockAttributes() == Trix.config.blockAttributes == {quote: Object, ... }  // true
getBlockAttributeNames() == ["quote", ...]  // true
```